kuma/static/styles/components/content.styl

```diff
@@ -7,6 +7,13 @@
 
 $table-blue = #d4dde4;
 
+/*
+    Used to draw borders around objects which should only be bordered in
+    the editor.
+*/
+
+$editor-box-border-style = solid 1px #6699FF40;
```

Comment:
Good idea -- although I don't like the idea of changing the style used for .hidden. It needs to have a unique look because it's a special case of showing you something that normal readers will not see. I'll work on this other stuff tomorrow though.